### DISCUSSION TITLE
Devicon for jq

### DISCUSSION THREAD

**Discussion by titaniumtraveler** [stedolan/jq#3178] [Ideas]
I've been writing a bit of jqlang lately and noticed that there isn't a devicon for it.
All it would really need would be a small icon (preferably as svg) and a [icon request](https://github.com/devicons/devicon/issues/new?assignees=&labels=request%3Aicon&projects=&template=icon_request.yml&title=%5BICON+REQUEST%5D%3A+icon+name) on the devicons repo.

**Comment by wader** [comment]
I guess the logo https://github.com/jqlang/jq/blob/master/docs/public/jq.svg is under same license as jq (does someone know?). If so then i think you could go ahead and send a request if you want. Where are the devicons used btw? ide's like vscode?

**Reply by titaniumtraveler** [reply]
> I guess the logo https://github.com/jqlang/jq/blob/master/docs/public/jq.svg is under same license as jq (does someone know?).

The docs state `jq is licensed under the MIT license (code) and the CC-BY-3.0 license (docs).`, so my guess is the icon is licensed under `CC-BY-3.0`.

> Where are the devicons used btw? ide's like vscode?

Yeah. They are also included in [nerdfonts](https://www.nerdfonts.com/), which makes it pretty convenient to use in terminal editors like nvim.

![image](https://github.com/user-attachments/assets/3ae90c5e-7508-4cac-bace-76975dd347fa)
